dev/bots/test.dart

```diff
@@ -528,6 +529,33 @@ Future<Null> _runFlutterAnalyze(String workingDirectory, {
   );
 }
 
+Future<Null> _verifyNoTestPackageImports(String workingDirectory) async {
+  final List<String> errors = <String>[];
+
+  new Directory(workingDirectory).listSync(recursive: true).where((FileSystemEntity entity) {
+    return entity is File && entity.path.endsWith('.dart');
+  }).map<void>((FileSystemEntity entity) {
+    final File file = entity;
+    final String data = file.readAsStringSync();
+    if (data.contains("import 'package:test/test.dart'") && !data.contains("// Defines a 'package:test' shim."))
+      errors.add(path.relative(file.path, from: workingDirectory));
+  });
+
+  // Fail if any errors
+  if (errors.isNotEmpty) {
+    print('$red━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━$reset');
+    final String s = errors.length == 1 ? '' : 's';
+    print('${bold}The following file$s depend on \'package:test\' directly:$reset\n');
+    print('\n');
+    print(errors.map((String line) => '  $line').join('\n'));
+    print('\n');
+    print('Rather than depending on \'package:test\' directly, use the shims provided.\n');
```

Comment:
Could you list the locations of the shims in the error message?

Reply:
done